src/middleware/fhir/router.js

```diff
@@ -0,0 +1,364 @@
+/* eslint-disable security/detect-object-injection */
```

Comment:
Copy and change this code from the node-fhir-server-core so it can support streaming